lib/rust/ensogl/core/src/data/color/data.rs

```diff
@@ -344,4 +346,18 @@ impl<C> Alpha<C> {
     pub fn mod_alpha<F: FnOnce(&mut f32)>(&mut self, f: F) {
         f(&mut self.alpha)
     }
+
+    /// Compute alpha overlay using alpha blending "over" operator. Assumes that both colors are
+    /// using straight alpha (not pre-multiplied).
```

Comment:
It's better to be clear here, since in the graphics pipeline we use `over` blending with premultiplied colors. That comment explains why this is different.